src/lib/Display.ts

```diff
@@ -61,7 +61,8 @@ export class UxDisplay implements Display {
 	}
 
 	public displayError(msg: string): void {
-		// This
+		// Setting "exit" to false means that the error will be displayed in a non-halting fashion instead of killing
+		// aborting the transaction entirely.
```

Comment:
Possible type: "killing aborting"... did you mean one or the other?

Reply:
Good catch.